hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/HRegion.java

```diff
@@ -1655,6 +1655,7 @@ public void setTimeoutForWriteLock(long timeoutForWriteLock) {
       }
     }
     this.closing.set(true);
+    LOG.info("Closing {}", this);
```

Comment:
@Joseph295 Can you change this to "Closing region {}"? `this` is anyways going to print region name as string only.

Reply:
> @Joseph295 Can you change this to "Closing region {}"? `this` is anyways going to print region name as string only.

I think the discussion above really makes sense, so do you guys think we add this log any more? maybe for debug level is enough? We only need it if we concern about the closing cost time for debug? Anyway, let me fix the format for that, any suggestion is welcome. Thanks

Reply:
Journal log is at Debug level anyways right? Let this be Info level, should be good. Imp point here is of timing of these logs and hence, if we debug from logs based on region name, we can see sorting of chronological events one after another and that way having this Info log is beneficial (although redundant).
Let's wait for @apurtell 's final call on this.